lib/stack.dart

```diff
@@ -33,4 +33,18 @@ class Stack<T> {
 
   /// get the length of the stack.
   int get length => size();
+  
+  // returns true if element is found at the top of the stack
+  bool find(T x) => (x == _list.last) ? true : false;
+  
+  // returns true if element is found in the stack
+  bool contains(T x) {
+    bool found = false;
+    for (var item in _list) {
+      if (x == item) {
+        found = true;
```

Comment:
@AunSyedShah you can use `return true;` here and `return false` by the end of the method and no need for the variable `found`

Reply:
done it, thanks friend.